toolchain/check/member_access.cpp

```diff
@@ -391,7 +391,6 @@ auto PerformMemberAccess(Context& context, Parse::NodeId node_id,
 
   return member_id;
 }
```

Comment:
Should not delete the newline that was there.

```suggestion
}

```